backend/migrations/versions/f24211d46708_set_password_and_phone_number_not_null.py

```diff
@@ -0,0 +1,30 @@
+"""set_password_and_phone_number_not_null
+
+Revision ID: f24211d46708
```

Comment:
Pleas adjust migration message to the normal format `Set password and phone number to not null`